C7/LogManager.cs

```diff
@@ -0,0 +1,68 @@
+using Godot;
+using Serilog;
+using Serilog.Templates;
+using Serilog.Templates.Themes;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Configuration;
+using Serilog.Formatting;
+using System;
+using System.IO;
+
+public class LogManager
+{
+	static LogManager()
```

Comment:
@pcen commented: "for any runtime initialisation that must be done before some feature is used, AFAIK we don't have an "entry point" for C7, so I think using a static factory class to ensure the initialisation happens first is the best way to solve this"

Reply:
I agree, at least for now. As mentioned in the other review if we're using a Godot sink that means the UI layer will have to be responsible for configuring the logger.